pulser/_seq_drawer.py

```diff
@@ -74,11 +74,16 @@ def gather_data(seq):
     return data
 
 
-def draw_sequence(seq):
+def draw_sequence(seq, time_slices=np.array([])):
     """Draw the entire sequence.
 
     Args:
         seq (pulser.Sequence): The input sequence of operations on a device.
+
+    Keyword args:
+        time_slices(numpy.ndarray): The time slices at wich the sequence is
+        to be sampled. If this is provided, the constant-by-part Sequence
+        defined on these slices is also drawn.
```

Comment:
A typo and indentation:
```suggestion
    Keyword args:
        time_slices(numpy.ndarray): The time slices at which the sequence is
            to be sampled. If this is provided, the constant-by-part Sequence
            defined on these slices is also drawn.
```